scene/gui/control.cpp

```diff
@@ -709,7 +709,7 @@ bool Control::can_drop_data(const Point2 &p_point, const Variant &p_data) const
 		}
 	}
 
-	return Variant();
+	return false;
```

Comment:
Good catch. I wonder if we should validate that the return value from the script instance is a `bool` on line 706 too?

It works as is because any `Variant` can convert to `bool`, but if there was a simple way to catch non-`bool` return values to raise a user-friendly error, that'd be good. Not related to this PR though (like this change :P).

Reply:
We could easily add a warning, but any Variant gets implicitly converted to bool, so wrong type doesn't bring any problems (other than maybe when someone doesn't know how some values are converted to bools).

Reply:
My point was that if you pass `Vector2(42, 0)` as return value in `can_drop_data()`, having a warning would be meaningful. But that's maybe a more general problem for any callback that accepts any Variant as return value even though it expects a specific type.